.github/workflows/update-changelog.yml

```
@@ -0,0 +1,89 @@
+            *)        LABEL="Change" ;;
+          esac
+
+          # Build the changelog entry
+          ENTRY="- **${LABEL} — ${DESC}** ([#${PR_NUMBER}](${PR_URL})):"
+
+          # Add PR body as sub-bullets if it has meaningful content
+          if [ -n "$PR_BODY" ] && ! echo "$PR_BODY" | grep -qiE "^($|generated|automatically|coderabbit)"; then
+            BULLETS=$(echo "$PR_BODY" | grep -E "^[-*] " | head -10 | sed 's/^[-*] /  - /')
+            if [ -n "$BULLETS" ]; then
+              ENTRY="${ENTRY}
+${BULLETS}"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)